src/moonblade/functions.rs

```diff
@@ -130,12 +130,14 @@ pub fn get_function(name: &str) -> Option<(Function, FunctionArguments)> {
         "last" => (last, FunctionArguments::unary()),
         "len" => (len, FunctionArguments::unary()),
         "log" => (
-            |args| unary_arithmetic_op(args, DynamicNumber::ln),
-            FunctionArguments::unary(),
-        ),
-        "log10" => (
-            |args| unary_arithmetic_op(args, DynamicNumber::log10),
-            FunctionArguments::unary(),
+            |args| {
+                match args.len() {
+                    1 => unary_arithmetic_op(args, DynamicNumber::ln),
+                    2 => binary_arithmetic_op(args, DynamicNumber::log),
+                    n => unreachable!("function \"log\" expected 1 or 2 arguments, but received {}", n),
```

Comment:
It causes build error without this line ie it wants me to address other cases apart from 1 and 2 arguments so I added this line

Reply:
Yes, that's because `match` clauses must be exhaustive. You could have written `_ => unreachable()` also. But this is also fine don't worry. I will merge soon.

Reply:
Okay. Thanks.